src/Init/Data/Ord.lean

```diff
@@ -59,6 +59,10 @@ instance : Ord USize where
 instance : Ord Char where
   compare x y := compareOfLessAndEq x y
 
+instance lexOrd [Ord α] [Ord β] : Ord (α × β) where
```

Comment:
I'd prefer if we didn't make this a global instance.  It conflicts with the general pointwise order instance which works more generally for preorders as well.

Reply:
Yeah, I thought we had discussed this instance before and was quite confused that it did not exist at all (but I may have confused it with the general Ord/LT diamond discussion).